pkg/controller/dataimportcron-controller_test.go

```diff
@@ -1596,6 +1623,7 @@ func newDataImportCron(name string) *cdiv1.DataImportCron {
 			Namespace:   metav1.NamespaceDefault,
 			UID:         types.UID(uuid.NewString()),
 			Annotations: map[string]string{},
+			Labels:      map[string]string{},
```

Comment:
Could only initialize the map in tests that care about them. Not such a big deal but this way other tests get to test a different input.